CHANGES.md

```diff
@@ -1,5 +1,29 @@
 # Changes
 
+## stable-2.13.4
+
+This stable release fixes a few issues in the proxy and in the outbound policy
+API. Two new configuration options are also introduced to configure the
+outbound (and inbound) cache discovery idle period for proxies. The
+configuration is supported through annotations and through Helm values.
+
+* Control Plane
+  * Fixed an issue where an HTTPRoute backendRef's namespace would be ignored
+  * Fixed an issue where probe paths with query params would not be authorized
+    through the default route
+  * Introduced outbound/inbound discovery cache idle timeout configuration
+    values through annotations
+  * Fixed bug where topology routing would not disable while service was under
+    load (thanks @MarkSRobinson!)
+* Proxy
+  * Fixed an issue where pods wouldn't be able to communicate with themselves
+    using their logical (Service) address
```

Comment:
in the edge changelog, this is worded as:
https://github.com/linkerd/linkerd2/blob/06a6f3be1dcabc38700f62d54487386fc2230987/CHANGES.md?plain=1#L52-L53
```suggestion
  * Fixed an issue where meshed pods could not communicate with themselves
    through a ClusterIP Service
```